standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -3712,16 +3712,17 @@ public long openTxn(String user) throws TException {
   }
 
   @Override
-  public long openTxn(String user, TxnType txnType) throws TException {
-    OpenTxnsResponse txns = openTxnsIntr(user, 1, null, null, txnType);
+  public long openTxn(String user, TxnType txnType, String dbUnderReplication) throws TException {
```

Comment:
are there any drawbacks of keeping the old method and not modifing every callsite by adding a new `null` argument?

Reply:
Yah! I have changed the methodology.